airflow/config_templates/airflow_local_settings.py

```diff
@@ -19,6 +19,8 @@
 
 import os
 
+from typing import Dict, Any
```

Comment:
How does this play out on Python2?

Reply:
Actually, can we put this behind the `if TYPE_CHECKING:` (or whatever the right if is)?

Reply:
This is basically the same in python 2 and 3. In python >=3.5, the `typing` module is part of the standard library; otherwise, we install the backport.